src/index.js

```diff
@@ -0,0 +1,190 @@
+import './styles.css';
+import './toggleTheme/styles.css';
+
+import { calculator } from './scripts/calculator';
+import { RemoveComputation } from './scripts/CommandPattern/RemoveComputation/RemoveComputation';
+import {
+  memorySaveButton,
+  memoryReadButton,
+  memoryClearButton,
+  memoryPlusButton,
+  memoryMinusButton,
+  buttonsNumbers,
+  buttonRemove,
+  buttonAnswer,
+  basicOperationButton,
+  pointButton,
+  changeSignButton,
+  squareButton,
+  cubedButton,
+  percentButton,
+  toPowersOfXButton,
+  xToPowerOfYButton,
+  computation,
+  mathSquareRootButton,
+  mathCubeRootButton,
+  xToRootOfYButton,
+  fractionButton,
+  factorialButton,
+} from './scripts/buttons';
+import { ChangeSignCommand } from './scripts/CommandPattern/ChangeSignCommand/ChangeSIgnCommand';
+import { PercentCommand } from './scripts/CommandPattern/PercentCommand/PercentCommand';
+import { MemorySaveCommand } from './scripts/CommandPattern/MemorySaveCommand/MemorySaveCommand';
+import { MemoryClearCommand } from './scripts/CommandPattern/MomoryClearCommand/MemoryClearCommand';
+import { MemoryReadCommand } from './scripts/CommandPattern/MemoryReadCommand/MemoryReadCommand';
+import { MemoryCalculate } from './scripts/CommandPattern/MemoryCalculateCommand/MemoryCalculateCommand';
+import { ExponentiationCommand } from './scripts/CommandPattern/ExponentiationCommand/ExponentiationCommand';
+import { MathRootCommand } from './scripts/CommandPattern/MarhRootCommand/MathRootCommand';
+import { DivideCommand } from './scripts/CommandPattern/DivideCommand/DivideCommand';
+import { FactorialCommand } from './scripts/CommandPattern/FactorialCommand/FactorialCommand';
+import { setComputation } from './scripts/functions/setComputation';
+import { checkOperator } from './scripts/functions/checkOperator';
+import {
+  calculationWithXandY,
+  count,
+} from './scripts/functions/calculationWithXandY';
+import { toggleTheme } from './toggleTheme/toggleTheme';
+
+toggleTheme();
+
+buttonRemove.addEventListener('click', () => {
+  calculator.setCommand(new RemoveComputation(computation.innerHTML));
+  calculator.executeCommand();
+});
+
+buttonAnswer.addEventListener('click', () => {
+  setComputation();
+});
+
+buttonsNumbers.forEach(button => {
+  button.addEventListener('click', () => {
+    basicOperationButton.forEach(button => {
+      button.disabled = false;
+    });
+    if (
+      computation.innerHTML[0] === '0' &&
+      computation.innerHTML.length === 1
+    ) {
+      return (computation.innerHTML = button.innerHTML);
```

Comment:
Так не делаем. Усложняешь код для чтения

```suggestion
      computation.innerHTML = button.innerHTML
      return computation.innerHTML
```